Change Log restructure

dbojicic opened the pull request:
Fixes for
- https://github.com/orgs/hl7au/projects/9/views/1?pane=issue&itemId=89225809
- https://github.com/orgs/hl7au/projects/9/views/1?pane=issue&itemId=89227571

Restructured the change log to organise entries under appropriate versions, updated formatting for consistency,

JayMurdoch commented on input/pagecontent/changes.md:
```diff
@@ -234,8 +165,8 @@ To help implementers, only the more significant changes are listed here.
           <li>add Person Recorded Sex Or Gender (<a href="https://jira.hl7.org/browse/FHIR-43834">FHIR-43834</a>)</li>
```
Change "Practitioner.extension added:" to "Practitioner.extension changed to:" to match other similar entries.

dbojicic replied:
Fixed

JayMurdoch replied:
Reviewed.

JayMurdoch commented on input/pagecontent/changes.md:
```diff
@@ -6,148 +6,79 @@
+    <li>Added invariant inv-obs-1 to preadopt R5 behaviour to ensure the BodyStructure Reference extension is present only when Observation.bodySite is not present (<a href="https://jira.hl7.org/browse/FHIR-47117">FHIR-47117</a>).</li>
   </ul>
 </li>
 <li>Changes to <a href="StructureDefinition-au-diagnosticresult.html">AU Base Diagnostic Result</a>:<ul>
```
Move up the list to sit below other "Changes to [specific profile]:" bullet points. Eg below AU Base Path Report.

dbojicic replied:
Done

JayMurdoch replied:
Reviewed.

JayMurdoch commented on input/pagecontent/changes.md:
```diff
@@ -251,8 +182,6 @@ To help implementers, only the more significant changes are listed here.
   <li>Changed Immunization.code to replace slices with Additional Bindings as 'preferred' in <a href="StructureDefinition-au-immunization.html">AU Base Immunisation</a> (<a href="https://jira.hl7.org/browse/FHIR-44821">FHIR-44821</a>).</li>
   <li>Added a guidance page on AU Base relationship with other HL7 AU FHIR implementation guides.</li>
```
I believe this should link to https://jira.hl7.org/browse/FHIR-43843 - please review and add if confirmed.

dbojicic replied:
Added

JayMurdoch replied:
Reviewed.

JayMurdoch commented on input/pagecontent/changes.md:
```diff
@@ -6,148 +6,79 @@
-    <li>Changed base definition to be <a href="StructureDefinition-au-diagnosticreport.html" >AU Base Diagnostic Report</a> (<a href="https://jira.hl7.org/browse/FHIR-46898">FHIR-46898</a>)</li>
-    <li>Changed Type for DiagnosticReport.basedOn from AU Base Diagnostic Request to AU Base Service Request (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>.)</li>
+    <li>Changed base definition to be <a href="StructureDefinition-au-diagnosticreport.html" >AU Base Diagnostic Report</a> (<a href="https://jira.hl7.org/browse/FHIR-46898">FHIR-46898</a>).</li>
+    <li>Changed type for DiagnosticReport.basedOn from AU Base Diagnostic Request to AU Base Service Request (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>).</li>
```
https://jira.hl7.org/browse/FHIR-46714 is referring to Diagnostic Request, but is included under Diagnostic Report. Please change.

dbojicic replied:
DiagnosticReport.based type was changed from a reference to AU Base Diagnostic Request to be a reference to reference to AU Base Service Request by FHIR-46714.
No change.

JayMurdoch replied:
Apologies, misread. No issue.

JayMurdoch commented on input/pagecontent/changes.md:
```diff
@@ -6,148 +6,79 @@
   <ul>
-    <li>Changed base definition to be <a href="StructureDefinition-au-diagnosticreport.html" >AU Base Diagnostic Report</a> (<a href="https://jira.hl7.org/browse/FHIR-46898">FHIR-46898</a>)</li>
-    <li>Changed Type for DiagnosticReport.basedOn from AU Base Diagnostic Request to AU Base Service Request (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>.)</li>
+    <li>Changed base definition to be <a href="StructureDefinition-au-diagnosticreport.html" >AU Base Diagnostic Report</a> (<a href="https://jira.hl7.org/browse/FHIR-46898">FHIR-46898</a>).</li>
```
Suggest changing wording to reflect the other base profile change below, eg "Changed Type for DiagnosticReport.basedOn from DiagnosticReport to AU Base Diagnostic Report"

dbojicic replied:
Changed

JayMurdoch replied:
My above comment was incorrect, however reviewed change to structure of this change entry to align with others.

JayMurdoch commented on input/pagecontent/changes.md:
```diff
@@ -6,148 +6,79 @@
   <ul>
-    <li>Changed base definition to be <a href="StructureDefinition-au-diagnosticreport.html" >AU Base Diagnostic Report</a> (<a href="https://jira.hl7.org/browse/FHIR-46898">FHIR-46898</a>)</li>
-    <li>Changed Type for DiagnosticReport.basedOn from AU Base Diagnostic Request to AU Base Service Request (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>.)</li>
+    <li>Changed base definition to be <a href="StructureDefinition-au-diagnosticreport.html" >AU Base Diagnostic Report</a> (<a href="https://jira.hl7.org/browse/FHIR-46898">FHIR-46898</a>).</li>
```
Suggest changing wording to reflect the other base profile change below, eg "Changed Type for DiagnosticReport.basedOn from DiagnosticReport to AU Base Diagnostic Report"

dbojicic replied:
Changed

JayMurdoch replied:
My above comment was incorrect, however reviewed change to structure of this change entry to align with others.

dbojicic commented on input/pagecontent/changes.md:
```diff
@@ -6,148 +6,79 @@
-  <li>Removed deprecated Encounter Description extension from <a href="StructureDefinition-au-encounter.html">AU Base Encounter</a> (<a href="https://jira.hl7.org/browse/FHIR-47121">FHIR-47121</a>)</li>
+  <li>Removed deprecated Encounter Description extension from <a href="StructureDefinition-au-encounter.html">AU Base Encounter</a> (<a href="https://jira.hl7.org/browse/FHIR-47121">FHIR-47121</a>).</li>
   <li>Changed Observation.effective[x] type in <a href="StructureDefinition-au-diagnosticresult.html"> AU Base Diagnostic Result </a> to remove type constraint (<a href="https://jira.hl7.org/browse/FHIR-48632">FHIR-48632</a>).</li>
 <li>Changes to <a href="StructureDefinition-au-imagingresult.html">AU Base Diagnostic Imaging Result</a>:<ul>
```
Done

JayMurdoch replied:
No change made, this group of changes is still at the end of the change log for this release.

dbojicic replied:
Moved

JayMurdoch replied:
It has moved to sit with the other grouped changes, but this change entry is sitting in between them now. Please move to the bottom:
- Changed Observation.effective[x] type in [AU Base Diagnostic Result ](https://github.com/hl7au/au-fhir-base/pull/StructureDefinition-au-diagnosticresult.html)to remove type constraint ([FHIR-48632](https://jira.hl7.org/browse/FHIR-48632)).

dbojicic replied:
Done

JayMurdoch replied:
Apologies - I missed before that this point was incorporated under AU Base Diagnostic Result batch and can now be deleted.

dbojicic replied:
removed

dbojicic commented on input/pagecontent/changes.md:
```diff
@@ -6,148 +6,79 @@
-    <li>Removed DiagnosticReport.category.coding:anatomicRegionOfInterest slice (<a href="https://jira.hl7.org/browse/FHIR-46933">FHIR-46933</a>)</li>
-    <li>Changed Type for DiagnosticReport.basedOn from AU Base Diagnostic Request to AU Base Service Request (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>.)</li>
+    <li>Removed DiagnosticReport.category.coding:anatomicRegionOfInterest slice (<a href="https://jira.hl7.org/browse/FHIR-46933">FHIR-46933</a>).</li>
+    <li>Changed type for DiagnosticReport.basedOn from AU Base Diagnostic Request to AU Base Service Request (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>).</li>
```
DiagnosticReport.based type was changed from a reference to AU Base Diagnostic Request to be a reference to reference to AU Base Service Request by FHIR-46714. 
No change.

JayMurdoch replied:
Apologies, misread. No issue.

dbojicic commented on input/pagecontent/changes.md:
```diff
@@ -6,148 +6,79 @@
-    <li>Changed base definition to be <a href="StructureDefinition-au-diagnosticreport.html" >AU Base Diagnostic Report</a> (<a href="https://jira.hl7.org/browse/FHIR-46898">FHIR-46898</a>)</li>
-    <li>Changed Type for DiagnosticReport.basedOn from AU Base Diagnostic Request to AU Base Service Request (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>.)</li>
+    <li>Changed base definition to be <a href="StructureDefinition-au-diagnosticreport.html" >AU Base Diagnostic Report</a> (<a href="https://jira.hl7.org/browse/FHIR-46898">FHIR-46898</a>).</li>
+    <li>Changed type for DiagnosticReport.basedOn from AU Base Diagnostic Request to AU Base Service Request (<a href="https://jira.hl7.org/browse/FHIR-46714">FHIR-46714</a>).</li>
```
DiagnosticReport.based type was changed from a reference to AU Base Diagnostic Request to be a reference to reference to AU Base Service Request by FHIR-46714.
No change.

dbojicic commented:
> See inline comments.

Thanks @JayMurdoch, ready for another review.

JayMurdoch commented on input/pagecontent/changes.md:
```diff
@@ -218,24 +150,24 @@ To help implementers, only the more significant changes are listed here.
+      <li>Added invariant inv-pra-3 to apply Australian Pronouns value set (<a href="https://jira.hl7.org/browse/FHIR-43719">FHIR-43719</a>).</li>
       <li>Added invariant inv-pra-4 to RSG.type to bind AU Recorded Sex or Gender Type (extensible) (<a href="https://jira.hl7.org/browse/FHIR-43834">FHIR-43834</a>).</li>
       <li>Added invariant inv-pra-5 to RSG.sourceDocument.type to bind AU Recorded Sex or Gender (RSG) Source Document Type (extensible) (<a href="https://jira.hl7.org/browse/FHIR-43834">FHIR-43834</a>).</li>
       <li>Added invariant inv-pra-6 to RSG.sourceDocument.type to bind Jurisdiction ValueSet - AU Extended (extensible) (<a href="https://jira.hl7.org/browse/FHIR-43834">FHIR-43834</a>).</li>
```
Commenting here for inline:
For entry "Changed context of extension [Australian Indigenous Status](https://github.com/hl7au/au-fhir-base/pull/922/StructureDefinition-indigenous-status.html) to add Person and RelatedPerson."
- add reference to github issue: https://github.com/hl7au/au-fhir-base/issues/794

dbojicic replied:
Done

JayMurdoch replied:
Reviewed.

JayMurdoch commented:
In addition to inline comments above, the following appear to be missing entries:
1. [FHIR-46282](https://jira.hl7.org/browse/FHIR-46282)
2. https://github.com/hl7au/au-fhir-base/issues/803
3. https://github.com/hl7au/au-fhir-base/issues/825

Please review and if agreed, add as entries.

JayMurdoch reviewed (changes requested):
See inline comments.

dbojicic commented:
> In addition to inline comments above, the following appear to be missing entries:
> 
> 1. [FHIR-46282](https://jira.hl7.org/browse/FHIR-46282)
> 2. [Remove max occurrence constraint on Australian Veteran Status extension definition #803](https://github.com/hl7au/au-fhir-base/issues/803)
> 3. [Add vaccine serial number to AU Base Immunization #825](https://github.com/hl7au/au-fhir-base/issues/825)
> 
> Please review and if agreed, add as entries.

1. Terminology resources were added and removed in the same release. No change.
2. Australian Veteran Status was added and updated in the same release, there is no need for a separate changelog entry for the update. No change. 
3. There is no impact on AU Base behaviour here; listing an extension as potentially useful in the introduction vs adding it as an optional does not change the behaviour of conformance requirements. No change.

dbojicic commented:
Thanks @JayMurdoch, ready for another review.

dt-r commented:
> In addition to inline comments above, the following appear to be missing entries:
> 
> 1. [FHIR-46282](https://jira.hl7.org/browse/FHIR-46282)
> 2. [Remove max occurrence constraint on Australian Veteran Status extension definition #803](https://github.com/hl7au/au-fhir-base/issues/803)
> 3. [Add vaccine serial number to AU Base Immunization #825](https://github.com/hl7au/au-fhir-base/issues/825)
> 
> Please review and if agreed, add as entries.

1. [FHIR-46282](https://jira.hl7.org/browse/FHIR-46282)  does not belong in the change log 
2. sure, that could be added
3. Isn't actually in AU Base Immunization - there is an outstanding Jira to do this, the GitHub migrated to Jira

JayMurdoch reviewed (changes requested):
One small change required.

dbojicic commented:
Thanks @JayMurdoch, ready for review.

JayMurdoch reviewed (changes requested):
Apologies Dusi - in that last change I missed that the entry had been duplicated. One final change required.
